MAUI/DataGrid/Column-types.md

```diff
@@ -1418,6 +1424,168 @@ The `DataGridNumericColumn` allows formatting the numeric data with culture-spec
+        </syncfusion:DataGridMultiColumnComboBoxColumn>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+var column = new DataGridMultiColumnComboBoxColumn()
+{
+    MappingName = "CustomerID",
+    HeaderText = "Customer ID",
+    ValueMember = "CustomerID",
+    DisplayMember = "CustomerID",
+    ItemsSource = viewModel.OrderDetails,
+    AutoGenerateColumns = false,
+    Columns = new ColumnCollection()
+    {
+        new DataGridTextColumn(){MappingName = "CustomerID"},
+        new DataGridNumericColumn(){MappingName = "ProductID"}
+    }
+};
+this.dataGrid.Columns.Add(column);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+SfDataGrid triggers, (CurrentCellDropDownSelectionChanged)[] event, when the SelectedValue is changed. (CurrentCellDropDownSelectionChangedEventArgs)[] of `CurrentCellDropDownSelectionChanged` event provides the information about the changed cell value.
+
+`SelectedIndex` property returns the index of selected item.
+`SelectedItem` property returns the selected item from drop down list.
+
+<img alt="DataGridMultiColumnComboBox column .NET MAUI DataGrid" src="Images\column-types\maui-datagrid-MultiColumn-ComboBox-column.png" width="404" height="396"/>
+
+### Auto-complete support
+
+You can allow SfMultiColumnComboBox to complete the entered input value automatically by setting the (AllowAutoComplete)[] property to `true`. `AllowAutoComplete` property will autocomplete the input value by comparing it to each item of the underlying data source of `DataGridMultiColumnComboBoxColumn` and returns the matched value by DisplayMember.
+
+### Filtering
+
+You can allow SfMultiColumnComboBox to filter the drop-down list items dynamically based on the text typed on editor by setting (AllowIncrementalFiltering)[] property to `true`. DataGridMultiColumnComboBoxColumn allows you to filter the items based on case by setting (AllowCaseSensitiveFiltering)[] to `true`. This will help users to select from large number of items.
+
+<img alt="DataGridMultiColumnComboBox column with Filtering .NET MAUI DataGrid" src="Images\column-types\maui-datagrid-MultiColumn-ComboBox-column-filtering.png" width="404" height="396"/>
+
+### Null value support
+
+You can allow null values in the column by setting the (AllowNullValue)[] property to `true`.
+
+N>
+The AllowNullValue will work only when the underlying property type is Nullable.
+
+### Popup Size Customization
+
+You can change the size of drop-down popup by setting (PopupWidth)[] and (PopupHeight)[] property. The PopupWidth will be set based on (PopUpMinWidth)[] property when the value is not set. By default, the `PopupMinWidth` value is 200.0.
+
+Similarly, the PopUpHeight will be set based on (PopUpMinHeight)[] when the value is not set. The default value of `PopUpMinHeight` is 300.0.
+
+SfMultiColumnComboBox can automatically adjust the popup width based on the actual size of SfDataGrid when setting (IsAutoPopupSize)[] to `true`.
```

Comment:
You can change the size of the drop-down popup by setting the PopupWidth and PopupHeight properties. If these values are not set, the popup width defaults to the PopupMinWidth property, which is 200.0 by default. Similarly, the popup height defaults to the PopupMinHeight property, which is 300.0 by default.
Additionally, SfMultiColumnComboBox can automatically adjust the popup width based on the actual size of the SfDataGrid by setting the IsAutoPopupSize property to true.